siaskynet/__init__.py

```diff
@@ -24,7 +24,8 @@ class SkynetClient():
         get_skykeys
     )
     from ._upload import (
-        upload_file, upload_file_request, upload_file_request_with_chunks,
```

Comment:
Please export `upload_file_request_with_chunks` as starting in `v2` we are guaranteeing backwards compatibility for the SDKs.

Reply:
exported